docs/run.js

```diff
@@ -1,26 +1,75 @@
 import {clickFeature, selectHighlightedFeatures} from './click_feature.js';
 import {sidebarDatasetsId, tableContainerId} from './dom_constants.js';
 import {drawTable} from './draw_table.js';
+import {showError} from './error.js';
 import {addLayer, addNullLayer, addScoreLayer, scoreLayerName, setMapToDrawLayersOn, toggleLayerOff, toggleLayerOn} from './layer_util.js';
 import {addLoadingElement, loadingElementFinished} from './loading.js';
 import {convertEeObjectToPromise} from './map_util.js';
 import {initializeAndProcessUserRegions} from './polygon_draw.js';
 import {setUserFeatureVisibility} from './popup.js';
-import processJoinedData from './process_joined_data.js';
-import {getScoreAsset} from './resources.js';
+import {processJoinedData} from './process_joined_data.js';
+import {getBackupScoreAsset, getScoreAsset} from './resources.js';
+import SettablePromise from './settable_promise.js';
 import {setUpToggles} from './update.js';
 
+export {createAndDisplayJoinedData, run};
+
+// For testing.
 export {
-  createAndDisplayJoinedData,
   drawTableAndSetUpHandlers,
-  run as default,
+  setScorePromiseAndReturnAssetName,
+  setScorePromises,
 };
 
 // Promise for score asset. After it's first resolved, we never need to download
 // it from EarthEngine again.
 let snapAndDamagePromise;
 const scalingFactor = 100;
 
+/**
+ * Contains `Promise<string>` with name of score asset. This is not just
+ * {@link getScoreAsset} because in the case that the asset with id
+ * {@link getScoreAsset} does not exist, we will fall back to the asset with id
+ * {@link getBackupScoreAsset}.
+ * @type {SettablePromise}
+ */
+const resolvedScoreAsset = new SettablePromise();
```

Comment:
Does this need to be a settable promise? I thought that was for when it was possible you would want to "get" a promise" before "setting" it and I think that won't happen here? But maybe good for safety?